spec/jobs/early_years_payments/practitioner_reminder_email_job_spec.rb

```diff
@@ -0,0 +1,241 @@
+require "rails_helper"
+
+RSpec.describe EarlyYearsPayments::PractitionerReminderEmailJob, type: :job do
+  let(:journey_configuration) { create(:journey_configuration, :early_years_payment) }
+
+  describe "#perform" do
+    subject(:perform_job) { described_class.new.perform }
+
+    context "when there are claims needing reminders" do
+      let(:eligible_ey_provider) { create(:eligible_ey_provider) }
+      let(:practitioner_email) { "practitioner@example.com" }
+
+      context "first reminder (1 week after provider submission)" do
+        let!(:claim_needing_first_reminder) do
+          claim = create(:claim, :submitted_by_provider, policy: Policies::EarlyYearsPayments)
+          claim.update!(practitioner_email_address: practitioner_email)
+          claim.eligibility.update!(
+            provider_claim_submitted_at: 8.days.ago,
+            practitioner_reminder_email_sent_count: 0,
+            practitioner_reminder_email_last_sent_at: nil
+          )
+          claim
+        end
+
+        let!(:claim_not_ready_for_first_reminder) do
+          claim = create(:claim, :submitted_by_provider, policy: Policies::EarlyYearsPayments)
+          claim.eligibility.update!(
+            provider_claim_submitted_at: 5.days.ago,
+            practitioner_reminder_email_sent_count: 0,
+            practitioner_reminder_email_last_sent_at: nil
+          )
+          claim
+        end
+
+        it "sends first reminder email after 1 week" do
+          expect {
+            perform_job
+          }.to have_enqueued_mail(EarlyYearsPaymentsMailer, :practitioner_claim_reminder)
+            .once
+        end
```

Comment:
I'd be tempted to change this to

```suggestion
  perform_enqueued_jobs { perform_job }
  expect(practitioner_email).to have_received_email("notify-template-id", complete_claim_url: "xxx")
```

that way the mailer logic runs so you can test that everything is wired up as expected. If you search the specs for `have_received_email` you'll see a bunch of example uses of this matcher. It's defined in `spec/support/have_received_email_matcher.rb`